coaster/utils.py

```diff
@@ -3,7 +3,7 @@
 from __future__ import absolute_import
 import six
 import time
-from datetime import datetime
+from datetime import datetime, date
```

Comment:
If `date` is only used in the test and not in the code, import it in the doctest.